docs/DeveloperGuide.md

```diff
@@ -126,8 +126,8 @@ The `UI` component,
 
 The `Model`,
```

Comment:
U updated the text for Read Only Task below but not the image ~ so do take note of this too :)